src/providers/bitcoin/BitcoinRPCProvider.js

```diff
@@ -109,6 +109,20 @@ export default class BitcoinRPCProvider extends JsonRpcProvider {
     return this.jsonrpc('getblockcount')
   }
 
+  async getAddressTransactions (address, start, end) {
+    const transactionIds = []
+    for (const blockNumber of _.range(start, end + 1)) {
+      const block = await this.getMethod('getBlockByNumber')(blockNumber, true)
```

Comment:
Can't we use `getAddressUtxos` for this? Wouldn't this be easier than checking each block?

Reply:
getAddressUtxos is a bitcore function. This provider assumes bitcoind. 
we could do with `listunspent` but you need to make sure you are watching the p2sh so i don't see much point in that.